qa/common/sequence_util.py

```diff
@@ -63,7 +63,7 @@
 
 _deferred_exceptions_lock = threading.Lock()
 _deferred_exceptions = None
-_jetson_slowdown_factor = 1.6
+_jetson_slowdown_factor = 6
```

Comment:
6 feels a little extreme. Add a comment above to revisit why jetson is 6x slower for trivial inferences.

Reply:
Will do.

Reply:
I think for most of the cases, changing the factor to 3 should be enough, but sometimes there is only one test case exceeding timeout, making the whole test to fail. Setting the factor to 6 can cover the largest test duration I got, but it is actually too much for general cases. Should I still set it to 6 and add a comment for explanation?

Reply:
Set it to 3 for now. I still feel 6x slowdown should not be observed on jetson unless due to machine specific issues. How often does the test fail when using 3 instead of 6?

Reply:
Checking.